targets/TARGET_STM/us_ticker_32b.c

```diff
@@ -23,33 +23,30 @@
 
 TIM_HandleTypeDef TimMasterHandle;
 
-static int us_ticker_inited = 0;
-
 void us_ticker_init(void)
 {
-    if (us_ticker_inited) return;
-    us_ticker_inited = 1;
-
-    TimMasterHandle.Instance = TIM_MST;
-
-    HAL_InitTick(0); // The passed value is not used
+    /* NOTE: assuming that HAL tick has already been initialized! */
```

Comment:
> I think that the above assumption is now ensured on all toolchains thanks to the recent rework on boot sequence - do you confirm ?

Yes, that is correct. As you are referring to mbed OS 5 boot sequence. 

mbed 2 should also be fine. We shall add a test for this to be certain that sdk is called prior C++ ctor